js/main.js

```diff
@@ -17,10 +23,115 @@ function storeValues(event) {
   data.entries.unshift(fieldEntries);
 
   data.nextEntryId++;
-
+  var fieldList = renderEntry(fieldEntries);
+  hiddenDiv.prepend(fieldList);
   placeHolder.setAttribute('src', './images/placeholder-image-square.jpg');
+  viewSwap('entries');
+  if (data.entries) {
+    isOn = false;
+    toggleNoEntries();
+  }
   $form.reset();
 
 }
 
 $form.addEventListener('submit', storeValues);
+
+function renderEntry(entry) {
+
+  var bullets = document.createElement('li');
+  bullets.classList.add('saved-entries');
+
+  var listRow = document.createElement('div');
+  listRow.classList.add('row');
+  bullets.appendChild(listRow);
+
+  var columnHalfOne = document.createElement('div');
+  columnHalfOne.classList.add('column-half');
+  listRow.appendChild(columnHalfOne);
+
+  var savedPhoto = document.createElement('img');
+  savedPhoto.classList.add('saved-photo');
+  savedPhoto.setAttribute('src', entry.url);
+  columnHalfOne.appendChild(savedPhoto);
+
+  var columnHalfTwo = document.createElement('div');
+  columnHalfTwo.classList.add('column-half');
+  listRow.appendChild(columnHalfTwo);
+
+  var nameTitle = document.createElement('h3');
+  nameTitle.classList.add('name-title');
+  var titleValueText = document.createTextNode(entry.title);
+  columnHalfTwo.appendChild(nameTitle);
+  nameTitle.appendChild(titleValueText);
+
+  var savedText = document.createElement('p');
+  savedText.classList.add('saved-text');
+  var notesValueText = document.createTextNode(entry.notes);
+  columnHalfTwo.appendChild(savedText);
+  savedText.appendChild(notesValueText);
+
+  return bullets;
+
+}
+
+function dataLoop() {
+  if (isOn === true) {
```

Comment:
This shouldn't be necessary. I'm not sure what the `isOn` variable is really for, the name doesn't really say its purpose.